### ISSUE TITLE
Look into showing data from month ago

### ISSUE THREAD

**Issue by ewels** [NationalGenomicsInfrastructure/NGI_dashboards#28]
Would be nice to show tick marks for previous month, as in the mockups. This is difficult though.

Came across [this](http://jsfiddle.net/jlbriggs/Hx3Gh/) - could work?

**Comment by remiolsen** [comment]
Yeah, this should work. I'm just thinking if we need to make some extra kpis in the updater script for this to work. The ones that are bound by 30 days (basically TaT) are just easy to fetch, however those that are more "snapshot" like probably more unstable and just turning the clock back 30 days on them might not be informative.

**Comment by ewels** [comment]
Yup, it was mostly the TaT that I was thinking of. Also maybe the queues? But it's not a big thing, and I'm not sure that I can do it on the plots yet.
